std_msgs/Header header
GeoPolygon polygon
